package.json

```diff
@@ -24,6 +24,7 @@
     "vue-template-compiler": "^2.6.10"
   },
   "dependencies": {
+    "netlify-cli": "^2.20.2",
```

Comment:
That's not a required dependency.

Reply:
I don't even use `netlify-cli`. We can put it in `dev-dependencies`.

Reply:
We don't even need `netlify-cli` in this repo tbh. It must've been the leftovers from previous codebase. Will remove it entirely.